app/views/view-messages.html

```diff
@@ -0,0 +1,86 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+{{ serviceName }} – Message centre
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <!-- Heading Content -->
+    <div class="govuk-grid-column-two-thirds">
+        <nav class="govuk-breadcrumbs" aria-label="Breadcrumb">
+            <ol class="govuk-breadcrumbs__list">
+                <li class="govuk-breadcrumbs__list-item">
+                    <a class="govuk-breadcrumbs__link" href="/">Home</a>
+                </li>
+                <li class="govuk-breadcrumbs__list-item">Message centre</li>
+            </ol>
+        </nav>
+        <h1 class="govuk-heading-l" style="margin-bottom: 10px;">
+            {{ claimantName }} vs {{ defendantName1 if data['applicationType'] == 'nonAutomatedMode' else
+            defendantName2 }}
+        </h1>
+        <p class="govuk-body" style="margin-bottom: 5px;">Case number: {{ caseNumber }}</p>
+        <p class="govuk-body">Property address: {{ propertyAddress }}</p>
+    </div>
+
+    <!-- Main Content -->
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
do you want to try making it govuk-grid-column-three-fourth and see if it looks better and fit better onto the page. I see the text all squeezed specially the hyperlinks